cmd/minishift/cmd/start.go

```diff
@@ -108,7 +109,12 @@ For the latter see 'minishift config -h'.`,
 		Run: runStart,
 	}
 
+	// enable experimental flags
+	//startCmd.Flags().BoolVar(&future, "future", false, "Enable experimental features (unsupported)")
+	future = os.Getenv("MINISHIFT_FUTURE") != ""
```

Comment:
hmm, what about 'MINISHIFT_EXPERIMENTAL=false minishift - h'? I would think we need to check for a "truish" value.